spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
- [ ] All the methods in the `User` Model should be tested, kindly make sure to implement unit tests for the `most_recent_posts` method